dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/OpenTelemetryProducer.java

```diff
@@ -65,11 +65,11 @@ public OpenTelemetry getOpenTelemetry() {
     private boolean checkEnabled(Map<String, String> oTelConfigs) {
         //In order to enable any of the tracing aspects, the configuration otel.experimental.sdk.enabled=true must be specified in any of the config sources available via MicroProfile Config
         if (oTelConfigs.get(ENV_ENABLE_PROPERTY) != null) {
-            if (oTelConfigs.get(ENV_ENABLE_PROPERTY).equals("true")) {
+            if (oTelConfigs.get(ENV_ENABLE_PROPERTY).equals("false")) {
                 return true;
             }
         } else if (oTelConfigs.get(CONFIG_ENABLE_PROPERTY) != null) {
-            if (oTelConfigs.get(CONFIG_ENABLE_PROPERTY).equals("true")) {
+            if (oTelConfigs.get(CONFIG_ENABLE_PROPERTY).equals("false")) {
                 return true;
             }
```

Comment:
These checks need reviewed. Setting `otel.sdk.disabled = False` would result in telemetry being disabled.

These are supposed to be loaded from MP Config, so we should follow MP Config's rules for converting to a boolean.

Reply:
This whole config bit should be looked at, let's raise a new issue for that.

Reply:
https://github.com/eclipse/microprofile-telemetry/blob/main/tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc#tracing-enablement

The spec states that it would be enabled with `otel.sdk.disabled = False`

Reply:
Yep, and that's the problem, with the current code it would be _disabled_ because `"False"` is not equal to `"false"`.

We'll raise a follow up issue to look again at this config part as we should be able to have MP Config do the conversion for us according to [it's rules](https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_built_in_converters).

For now, can you change this code to use [Boolean.valueOf(String s)](https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#valueOf(java.lang.String)) to do the conversion?